### PR TITLE
feat: Changes related to openshift-ci onboarding

### PR DESCRIPTION
**Pull request by tisutisu**
This PR contains changes required for on boarding openshift-ci to the image-controller repo and is part of the story https://issues.redhat.com/browse/STONEBLD-1101

### CODE REVIEW

File: hack/preview.sh [hunk 1]
```diff
@@ -186,6 +186,7 @@ sed -i.bak "s/rekor-server.enterprise-contract-service.svc/$rekor_server/" $ROOT
 
 [ -n "${IMAGE_CONTROLLER_IMAGE_REPO}" ] && yq -i e "(.images.[] | select(.name==\"quay.io/redhat-appstudio/image-controller\")) |=.newName=\"${IMAGE_CONTROLLER_IMAGE_REPO}\"" $ROOT/components/image-controller/base/kustomization.yaml
 [ -n "${IMAGE_CONTROLLER_IMAGE_TAG}" ] && yq -i e "(.images.[] | select(.name==\"quay.io/redhat-appstudio/image-controller\")) |=.newTag=\"${IMAGE_CONTROLLER_IMAGE_TAG}\"" $ROOT/components/image-controller/base/kustomization.yaml
+[[ -n "${IMAGE_CONTROLLER_PR_OWNER}" && "${IMAGE_CONTROLLER_PR_SHA}" ]] && yq -i e "(.resources[] | select(. ==\"*github.com/redhat-appstudio/image-controller*\")) |= \"https://github.com/${IMAGE_CONTROLLER_PR_OWNER}/build-service/config/default?ref=${IMAGE_CONTROLLER_PR_SHA}\"" $ROOT/components/image-controller/base/kustomization.yaml
```
Comment (Michkov) [inline hack/preview.sh:189]: `build-service` ?

Comment (tisutisu) [reply hack/preview.sh]: Thanks for pointing it out. Changed it now.

**Review by Michkov — APPROVED** [review]

**Comment by psturc** [comment]
```
"message": "The step \"upload-sbom\" in TaskRun \"integration-sui16521f44ef761119b64a2b66233e93d1-build-container\" failed to pull the image \"\". The pod errored with the message: \"Back-off pulling image \"quay.io/redhat-appstudio/cosign@sha256:18b3716a6225727877475e1ab4f2493915e72cffd2ce431e9901d2ed2e4b2c0b\".\"",
```

**Pull request merged**